data/campaigns/Eastern_Invasion/scenarios/04a_An_Elven_Interlude.cfg

```diff
@@ -278,11 +279,13 @@
         [/unit]
         [message]
             speaker=Nafga
+            #po: What are your orders for me to perform, Addogin?
             message= _ "And what about me?"
         [/message]
         [message]
             speaker=Addogin
             #po: "you" being a single male Orcish Slayer or Orcish Nightblade
+            #po: later in the campaign, it's revealed that the necromancers are buying corpses, and a lot are being sold
```

Comment:
It looks like something not useful for translation at this time and that would duplicate the work or leave inconsistencies if the referenced part was changed